index.html

```diff
@@ -1,12 +1,176 @@
+          </li>
+
+          <li class="advantages-item">
+            <h3 class="advantages-title">Небольшая площадь</h3>
+            <!--Здесь д.б. линейка-->
+            <p>Все достопримечательности находятся очень близко</p>
+          </li>
+          <li class="advantages-item">
+            <h3 class="advantages-title">Красивая дорога</h3>
+            <!--Здесь д.б. линейка-->
+            <p>Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+          </li>
+          <li class="advantages-item">
+            <h3 class="advantages-title">Мало туристов</h3>
+            <!--Здесь д.б. линейка-->
+            <p>Большинство едет в Гранд Каньон и толпится там</p>
+          </li>
+
+          <!--Выворотка на синем-->
+          <li class="advantages-item">
+            <!--Здесь д.б. фото-->
+            <img class="foto-left" src="" width="800" height="384" alt="Мост дьявола">
+            <h3 class="advantages-title">Там есть мост дьявола</h3>
+            <!--Здесь д.б. линейка-->
+            <p>Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+          </li>
+        </ul>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+        <ul>
+          <li class="advantages-item">
+            <!--Здесь д.б. картинка домик-->
+            <h3 class="advantages-title">Жильё</h3>
+            <p>Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+          </li>
+          <li class="advantages-item">
+            <!--Здесь д.б. картинка бургер-->
+            <h3 class="advantages-title">Еда</h3>
+            <p>Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+          </li>
+          <li class="advantages-item">
+            <!--Здесь д.б. картинка коробочка-->
+            <h3 class="advantages-title">Сувениры</h3>
+            <p>Не только китайского, но и настоящего местного производства!</p>
+          </li>
+        </ul>
+      </section>
 
+      <section class="search">
+        <h2>Заинтересовались?</h2>
+        <p class="subtitle">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне
+        </p>
+        <!--Кнопка "Поиск гостиницы в Седоне"-->
+        <a class="navigation-link" href="#">
+          <span class="button-big">Поиск гостиницы в Седоне</span>
+        </a>
+      </section>
+
+      <section class="newsletter">
+        <h2>Подпишитесь на рассылку</h2>
+        <p class="subtitle">Только полезная информация и никакого спама, честное бойскаутское!</p>
+        <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label for="newsletter-email"></label>
+          <input class="field" placeholder="Ваш e-mail" type="email" name="newsletter-email" id="newsletter-email"
+            required>
+          <button class="newsletter-button" type="button">
```

Comment:
```suggestion
          <button class="newsletter-button" type="submit">
```
Такой тип должен быть у кнопки отвечающей за отправку данных из формы.